vnext/Microsoft.ReactNative/IReactContext.idl

```diff
@@ -47,6 +48,9 @@ The subscriptions added to the `IReactInstanceSettings.Notifications` are kept a
     DOC_STRING("Get `ReactDispatcherHelper::JSDispatcherProperty` from the Properties property bag.")
     IReactDispatcher JSDispatcher { get; };
 
+    DOC_STRING("Get the JSI runtime for the running React instance. It can be null if Web debugging is used.")
+    JsiRuntime JsiRuntime { get; };
```

Comment:
We should start versioning new properties/methods/types.

Reply:
Why?  Isn't that just code bloat and overhead at this point.  We haven't done any binary distribution yet.

Reply:
I agree with Andrew: at this point producing all these version interfaces sounds as an overhead. We will start doing when we start having the binary distributions.